test/environmentSettings.ts

```diff
@@ -71,6 +76,8 @@ export class EnvironmentSettings {
       : this._logLevel;
     this._javaHome = process.env.JAVA_HOME || this._javaHome;
     this.useExistingProject = process.env.USE_EXISTING_PROJECT_PATH;
```

Comment:
@mingxuanzhangsfdx did you want to get rid of `USE_EXISTING_PROJECT_PATH` in favor of `LOCAL_PROJECT_PATH`. If not can you justify having both?

Reply:
Yes I do. I want to replace with `LOCAL_PROJECT_PATH` for better clarification. Just updated